workers/unity/Packages/com.improbable.gdk.mobile/Worker/MobileWorkerConnector.cs

```diff
@@ -0,0 +1,20 @@
+using Improbable.Gdk.Core;
+
+namespace Improbable.Gdk.Mobile
+{
+    public abstract class MobileWorkerConnector : WorkerConnector
```

Comment:
Wait why do we have a class tree like:

`AbstractMobileWorkerConnector` > `MobileWorkerConnector` > `WorkerConnector` ? 

Can't the mobile ones just be wrapped together into one?

Reply:
The idea is that `MobileWorkerConnector` is the core mobile class, part of the mobile package while `AbstractMobileWorkerConnector` is our Playground implementation and it exists because we have overlapping logic for Android and iOS.

Reply:
Hmmmm I see - I would suggest rethinking the name of the one in Playground as its super confusing at the moment

Reply:
Don't name the more derived one to sound like it's less derived.

Reply:
yeah, this doesn't make too much sense... 
as people might only use the iOS worker or the android worker instead of both, I would suggest just duplicating the code to make it easier to understand.

Additionally, all the splash screen stuff can be moved to it's own game object

Reply:
Guessing this conversation can probably be resolved